zuul/connection/github.py

```diff
@@ -521,51 +526,57 @@ def getRepoPermission(self, owner, project, login):
         return perms.json()['permission']
 
     def commentPull(self, owner, project, pr_number, message):
-        pull_request = self.github.issue(owner, project, pr_number)
+        github = self.getGithubClient()
+        pull_request = github.issue(owner, project, pr_number)
```

Comment:
Why, if we have the owner and project info, are we creating a github client without that info? Wouldn't it be more appropriate to create the client in the scope of the owner/project, so that we use the integration key to comment?